src/js/components/MaskedInput/MaskedInput.js

```diff
@@ -485,9 +485,7 @@ const MaskedInput = forwardRef(
                   // Determine whether the label is done as a child or
                   // as an option Button kind property.
                   const child = !theme.button.option ? (
-                    <Box pad={{ horizontal: 'small', vertical: 'xsmall' }}>
```

Comment:
same comment from here https://github.com/grommet/grommet/pull/7748/files#r2260877763

what you think?

Reply:
Oh yes good call, tagging with the discussion label

Reply:
just add comment we do not need for hpe theme just flag if we need future wise

Reply:
wonder if we should align on the comments 

I had 
```
        // We are not going to add a theme object now since HPE
                    // theme is not needed, but in the future we may want
                    // to support a theme prop.
```

but open to yours as well just thinking to have it the same across all

Reply:
@jcfilben 

```
// Not adding a theme object now because this code path
// is not used in the HPE theme, but we may add theme
// support here in the future.
```

Reply:
Changed in latest commit